### PR TITLE
Feat/ade 495/generic macro structured data

### PR DESCRIPTION
**Pull request by SamMurphyDev**

### CODE REVIEW

File: src/main/java/au/com/agiledigital/idea_search/macros/StructuredField.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  private PageBuilderService pageBuilderService;
+
+  public StructuredField(
+    @ComponentImport PageBuilderService pageBuilderService
+  ) {
+    this.pageBuilderService = pageBuilderService;
+  }
+
+  private String render(StructuredCategory category, String body) {
+    Map<String, Object> test = new HashMap<>();
```
Comment (robinMcA) [inline src/main/java/au/com/agiledigital/idea_search/macros/StructuredField.java:24]: update name from test

**Review by robinMcA — COMMENTED** [review]
Just the `test` var, other wise looks good to me

File: src/main/java/au/com/agiledigital/idea_search/macros/StructuredField.java (hunk 1 above)
Comment (SamMurphyDev) [reply src/main/java/au/com/agiledigital/idea_search/macros/StructuredField.java]: Changed

**Review by robinMcA — APPROVED** [review]
Looks good

**Pull request merged**